test/big_logic_value_test.dart

```diff
@@ -74,6 +74,31 @@ void main() {
         expect(bblv > fblv, LogicValue.zero);
         expect(fblv > bblv, LogicValue.one);
       });
+      test('big math len=$len', () {
+        expect(sslv + fslv, LogicValue.ofInt(2, len));
+        expect(sslv - fslv, LogicValue.ofInt(6, len));
+        expect(fslv - sslv, LogicValue.ofInt(-6, len));
+
+        expect(sslv * fslv, LogicValue.ofInt(-8, len));
+
+        expect(sslv + fslv, LogicValue.ofBigInt(BigInt.from(2), len));
+        expect(sslv - fslv, LogicValue.ofBigInt(BigInt.from(6), len));
+        expect(fslv - sslv, LogicValue.ofBigInt(BigInt.from(-6), len));
+
+        expect(fslv * sslv, LogicValue.ofBigInt(BigInt.from(-8), len));
+
+        expect(sblv + fblv, LogicValue.ofInt(2, len));
+        expect(sblv - fblv, LogicValue.ofInt(6, len));
+        expect(fblv - sblv, LogicValue.ofInt(-6, len));
+
+        expect(sblv * fblv, LogicValue.ofInt(-8, len));
+
+        expect(sblv + fblv, LogicValue.ofBigInt(BigInt.from(2), len));
+        expect(sblv - fblv, LogicValue.ofBigInt(BigInt.from(6), len));
+        expect(fblv - sblv, LogicValue.ofBigInt(BigInt.from(-6), len));
+
+        expect(fblv * sblv, LogicValue.ofBigInt(BigInt.from(-8), len));
```

Comment:
Nice, what about division and modulo though?

Also, with these tests perhaps we can remove the "WARNING: Signed math is not fully tested." messages in logic_value.dart?

Reply:
division doesn't seem to work properly for negative numbers.
8 / -2  should be -4, but it is 0.
I felt I needed to understand Dart ints and LogicValue a bit more before I figure out the problem.  
Currently this is not a big/little problem, but an int problem.

Reply:
Ok, got it, I filed a separate issue to track here so we're not hung up on this PR
https://github.com/intel/rohd/issues/411